### ISSUE TITLE
Update /check API to approve address of deleted website

### ISSUE THREAD

**Issue by leafes** [nostrband/nostr-site-manager#88]
details https://jam.dev/c/6f26ffd3-7193-4ac7-8736-5ac6f61626fb

**Comment by leafes** [comment]
possibly we need to pass website url to "domain" field on /delete call
 ![Image](https://github.com/user-attachments/assets/356bb4f3-3d49-4380-83f7-447e9edd0e54

**Comment by nostrband** [comment]
> possibly we need to pass website url to "domain" field on /delete call, to delete it from the DB

User may have anything in their website url (event can be edit outside of our UI), we're looking for the site by siteId - only one domain can be assigned to siteId (siteId = naddr of site event kind 30512). 

This issue can be closed, whatever problems with deletes we have can't be solved with adding 'domain' here.

**Comment by nostrband** [comment]
On the video you couldn't re-assign domain of the deleted site, right? That's a problem too. You should be able to reuse the domain name of your deleted site.

**Comment by leafes** [comment]
@nostrband yep, i delete [at97.npub.pro](http://at97.npub.pro/) website from dashboard and then, when i create and edit new website settings: 

1. /check API validate at97.npub.pro as already used 
2. I can bypass /check validation by clicking "save" before server response

**Comment by nostrband** [comment]
Check-bypass should be a separate issue for @a-fralou

**Comment by nostrband** [comment]
> 1. /check API validate at97.npub.pro as already used

Should be fixed now.

Also please:
> Check-bypass should be a separate issue for @a-fralou

**Comment by leafes** [comment]
@nostrband 

> Check-bypass should be a separate issue for @a-fralou
i separated an issue for this: 
https://github.com/nostrband/nostr-site-manager/issues/91

>Should be fixed now.

now i'm unable to delete website, i receive either 400 or 500 status on /delete call.
<details>
  <summary>Screenshots</summary>
  
<img width="1792" alt="Screenshot 2024-10-14 at 18 44 54" src="https://github.com/user-attachments/assets/67105dee-7b72-4dda-9639-1f85b74ee4ef">
<img width="1520" alt="Screenshot 2024-10-14 at 18 44 25" src="https://github.com/user-attachments/assets/91a40fbd-a0cb-4868-8dc8-0e5cc116cc8e">
<img width="1520" alt="Screenshot 2024-10-14 at 18 44 23" src="https://github.com/user-attachments/assets/47620425-2441-4cc0-a5a2-d345bd6b8d29">
<img width="1520" alt="Screenshot 2024-10-14 at 18 43 53" src="https://github.com/user-attachments/assets/a5f0d434-52c8-40ac-8f77-e7e62f940475">
<img width="1520" alt="Screenshot 2024-10-14 at 18 43 51" src="https://github.com/user-attachments/assets/b74fbcea-1da1-4c8c-84b9-8acd601c5c98">

</details>

**Comment by nostrband** [comment]
Can you please try again now?

**Comment by leafes** [comment]
Fixed. The domain of the deleted website is available to take.

> Can you please try again now?
